tools/kubernetes/shuffle-file-cleaner/src/main/scala/org/apache/kyuubi/tools/kubernetes/KubernetesShuffleFileCleaner.scala

```diff
@@ -0,0 +1,168 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.tools.kubernetes
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+
+import org.apache.kyuubi.Logging
+import org.apache.kyuubi.tools.kubernetes.Constants._
+
+object KubernetesShuffleFileCleaner extends Logging {
```

Comment:
This approach has a strong dependency with Apache Spark block manager, so it's better to show the shuffle file structure at class description: 
```
local-dir/
  blockmgr-uuid/
    hash-sub-dir/
      shuffle-data
      shuffle-index
```